pkg/term/windows/windows_test.go

```diff
@@ -0,0 +1,3 @@
+// Dummy file
```

Comment:
@tiborvass this is needed otherwise `go test ./...` fails on janky linux boxes b/c this package is the only fully-non-linux package ever and compiler can't pick up any files due to build tags and the build fails. we discussed this on IRC, many agreed. 

@brendandixon I remember we decided to call this `dummy.go` or something, but there are 2 empty files as `_test` in pkg/term/windows?